install/kubernetes/cilium/values.yaml

```diff
@@ -1777,6 +1778,30 @@ tls:
     # -- Generated certificates validity duration in days. This will be used for auto generated CA.
     certValidityDuration: 1095
 
+  # -- Configure the CA trust bundle used for the validation of the certificates
+  # leveraged by hubble and clustermesh. When enabled, it overrides the content of the
+  # 'ca.crt' field of the respective certificates, allowing for CA rotation with no down-time.
+  caBundle:
+    # -- Enable the use of the CA trust bundle.
+    enabled: false
+
+    # -- Name of the ConfigMap containing the CA trust bundle.
+    name: cilium-root-ca.crt
```

Comment:
Resource names cannot contain dots.

Reply:
The [official Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) states *"Most resource types require a name that can be used as a DNS subdomain name as defined in [RFC 1123](https://tools.ietf.org/html/rfc1123)"*, allowing '-' and '.' in addition to alphanumeric characters. I've selected `cilium-root-ca.crt` as name to mimic the `kube-root-ca.crt` configmap created by default in recent versions of Kubernetes, but I'm open to change it if you feel a different one could be better.

Reply:
Ah, that's fine, I had forgotten dots were allowed